### DISCUSSION TITLE
Plugin doesn’t work since i upgraded to Grafana 8

### DISCUSSION THREAD

**Discussion by seb201** [grafana/grafana-json-datasource#134] [Q&A] [answered]
Hello,

today i upgrade my Grafana installation to version 8. Now this Plugin doesn't work anymore.
It say: Error: Fetch error: 404 Not Found Instantianing.
I reinstall the plugin but with no anther results. 

I hope you can help me.

Sincerely Sebastian 

![1AA06A70-FFAC-4AF9-A095-05F85966684B](https://user-images.githubusercontent.com/35576062/122672594-964b8f80-d1cc-11eb-8590-6c517f4a2ded.jpeg)

**Comment by marcusolsson** [comment]
I think it might be a permission issue. Could you check and see if this works for you as well?:

https://twitter.com/foosel/status/1404097464278339589?s=21

**Comment by seb201** [comment, marked as answer]
I type in: **cd /var/lib/grafana** and **chmod -R o-x plugins** and now I works, Thank you!!!
